### ISSUE TITLE
Is it possible to do a heading without creating a card?

### ISSUE THREAD

**Issue by davidh38** [eyeinsky/org-anki#102]
I would like to categorize  my questions. Is it possible to do a heading 1 without creating a card out of it?

Example:
* Heading 1
** Heading 2
answer 2

Heading one should not create a card. Just Heading 2 and answer 2 should be a new card.

**Comment by eyeinsky** [comment]
Hi @davidh38, yes, that is how it works! You can create a card from any single heading/content pair with `org-anki-sync-entry`.

**Comment by davidh38** [comment]
@eyeinsky I appreciate your work, thanks a lot btw.

No, I want the Heading 1 not to create a card, it is just a heading, but when I sync it, it creates an entry, but it should not, because it is only a heading and thus a categorisation for my actual cards.

**Comment by eyeinsky** [comment]
What command do you use to create the card and where is your cursor when doing it? If you are within Heading 2 and run `org-anki-sync-entry` then it should just create a card where front is "Heading 2" and back is "answer 2".

**Comment by davidh38** [comment]
I run org-anki-sync-all with. If I use org-anki-sync-entry, I would have to do it for a lot of entries.

I just would like to run org-anki-sync-all and not have heading 1 create a card entry and not do  org-anki-sync-entry for each of my approximately over 100 entries. Is that possible or can I not use headings for cards categorization?

**Comment by eyeinsky** [comment]
Yes, using the sync-all would make cards of all headings. I guess the improvement here would be to skip all headings that have no textual content.

The other way to do it at the moment would be to do all entries manually at first (i.e, use `org-anki-sync-entry`) and then use `org-anki-update-all` afterwards -- this would only sync again all cards which have been synced previously.

**Comment by davidh38** [comment]
Hello @eyeinsky,

is there any other way to have a delimiter, I don't want to sync everything like this, but in bulk. Is there some other delimiter, I could sue for example
# this
or
---------
because I would like to categorise by deck. How do you manage this categorisation?

**Comment by eyeinsky** [comment]
Github markup is making it hard to follow -- could you in-between code markers (three backticks) show what you mean? E.g

```
* Heading 1
** This I'd like to be card front
this I'd like to be the card back
---
this shouldn't be part of card back anymore, it's delimited by the "---" on the line above.
```

**Comment by davidh38** [comment]
@eyeinsky kk, will try it "---" would work great, because than I could make it visible, which deck is which. Thank you.

**Comment by davidh38** [comment]
--- does not work at all, because it is part of the card. How do you categorize the cards, when you have more than one deck or are you syncing each care one by one the first time?

edit: I now sync each one seperately and then use update all for now

**Comment by eyeinsky** [comment]
Hi, I may have solved your problem, please test latest master/v3.3.1!

What I do is I filter out all headings which have no immediate content (immediate in the sense that they only have sub-headings, but no body text).

**Comment by 1over137** [comment]
It does not appear to work, cards are still made for the headings.

**Comment by eyeinsky** [comment]
Right, thanks for testing! Could you please provide a small sample org file for me to test this as well?

**Comment by 1over137** [comment]
This should be enough:
```
#+title: Emacs keybinds
#+ANKI_DECK: emacs
* Buffers and window management
** show buffers in workspace
SPC b b
** c2::Show messages tab
C-`
```

On a different note, would it be possible to support adding context for the card? In this case it would be the parent heading. Maybe we can have a file-local setting that says the heading level to be considered as notes, and all the parent headings should be considered as the context.
